test-requirements.txt

```diff
@@ -7,11 +7,13 @@ pytest-random-order
 nbsphinx
 sphinx_rtd_theme
 mypy==1.5.1
+types-mock
 types-python-dateutil
 types-requests
 types-paramiko
 mpi4py
 
+
```

Comment:
might as well tidy this up when addressing the value error comment

Reply:
Fixed.